test/distributed/org/apache/cassandra/distributed/test/cql3/StatefulASTBase.java

```diff
@@ -333,11 +333,18 @@ protected ConsistencyLevel mutationCl()
         {
             var inst = selectInstance(rs);
             String postfix = "on " + inst;
+            if (mutation.isCas())
+            {
+                postfix += ", would apply " + model.shouldApply(mutation);
+                // CAS doesn't allow timestamps
```

Comment:
CAS actually rejects mutations, so had to add this logic.

I was under the impression that Accord Interop allows timestamps due to paxos allowing, but doesn't look like thats 100% true?